activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -164,8 +164,26 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-donald-e-knuth">Donald E. Knuth</a>
-  def parameterize(sep = '-'.freeze)
-    ActiveSupport::Inflector.parameterize(self, sep)
```

Comment:
Interestingly, it turns out that we never had documentation about changing the separator, can you add that also? :)